src/OrchardCore.Modules/OrchardCore.Users/Drivers/RegistrationSettingsDisplayDriver.cs

```diff
@@ -0,0 +1,31 @@
+using System.Threading.Tasks;
+using OrchardCore.DisplayManagement.ModelBinding;
+using OrchardCore.DisplayManagement.Views;
+using OrchardCore.Entities.DisplayManagement;
+using OrchardCore.Settings;
+using OrchardCore.Users.Models;
+
+namespace OrchardCore.Users.Drivers
+{
+    public class RegistrationSettingsDisplayDriver : SectionDisplayDriver<ISite, RegistrationSettings>
+    {
+        public const string GroupId = "registrationSettings";
+
+        public override IDisplayResult Edit(RegistrationSettings section)
+        {
+            return Shape<RegistrationSettings>( "RegistrationSettings_Edit", model => {
```

Comment:
You must have a global setting that adds spaces inside all parenthesizes ;)

Reply:
Yes, easier to read for me. :)